### DISCUSSION TITLE
Bitcoin Design YouTube channel

### DISCUSSION THREAD

**Discussion by GBKS** [BitcoinDesign/Guide#126] [Archived]
We have started creating various videos that currently don't have a central home. Examples (also see this [playlist](https://www.youtube.com/playlist?list=PL4hsXZYKTCz15guoMZUumWVlL79NlrXYO)):
- Recordings of design review calls by @ConorOkus 
- How-to propose changes videos by @pavlenex 
- FOSDEM conference presentation by a group of us
- Guide status updates by @GBKS 
- Abstractions video update by @johnsBeharry 

At the moment, everyone uploads them on their channels, making them hard to find. Video backups are also up to each individual.

A potential solution is to create a dedicated YouTube channel or Bitcoin Design, managed by core contributors. This could also be used to create and manage playlists with good Bitcoin design content (created by community members and others).

Considering that crypto content is sometimes taken down by YouTube, a backup strategy would be great. Not sure what that could look like at the moment.

For testing, I created a new YouTube channel we can use. It's a brand channel, which allows for multiple managers. Getting a custom URL (youtube.com/bitcoindesign) requires 100 subscribers and 30 days channel age.

**Comment by pavlenex** [comment]
We touched a bit on this topic on yesterday's informal call. 

@GBKS If you have the account set up, let's try to add members there so that videos can be uploaded. For backups, we can either use a shared cloud account or just upload content to a separate repo (so that heavy files don't make cloning a repo impossible. Perhaps just a YT channel that will host these as private. I know @johnsBeharry had an idea about using some sort of decentralized youtube so might be worth exploring as a backup option.

**Reply by GBKS** [reply]
Invites sent.

It is not possible to move videos from one account to another. How about we just upload new videos to that channel and create a playlist for all the old videos? That way we don't have to re-upload them and update old links.

**Reply by johnsBeharry** [reply]
@pavlenex not decentralised youtube - [youtube-dl](https://github.com/ytdl-org/youtube-dl) which is a script that can be used to download videos uploaded to the yt account, then uploaded Netlify which would act as our CDN (100gb bandwidth and I believe storage also).

A video repo using Git may end up being 10s of GB and Git and Github are not suitable for large files.

> To ensure performance and reliability for our users, we actively monitor signals of overall repository health. Repository health is a function of various interacting factors, including size, commit frequency, contents, and structure.
> 
> We recommend repositories remain small, ideally less than 1 GB, and less than 5 GB is strongly recommended. Smaller repositories are faster to clone and easier to work with and maintain. Individual files in a repository are strictly limited to a 100 MB maximum size limit. For more information, see "Working with large files."

> We recommend repositories remain small, ideally less than 1 GB, and less than 5 GB is strongly recommended

> Git is not designed to serve as a backup tool.

https://docs.github.com/en/github/managing-large-files/working-with-large-files

**Reply by pavlenex** [reply]
> How about we just upload new videos to that channel and create a playlist for all the old videos?

My videos are unlisted, I would like to upload them to the channel and will take care of link replacing (since they are only in a #125). Tutorials like this are probably evergreen and should be indexable. Are you guys okay with me uploading those 3 videos to the channel?

lol, for some reason we need to verify a phone number to add custom thumbnails :laughing: 
![Screenshot from 2021-01-23 11-55-04](https://user-images.githubusercontent.com/36959754/105576266-f0f1d400-5d71-11eb-8692-7c71f7cddcad.png)

**Reply by GBKS** [reply]
Just verified it.

**Comment by ConorOkus** [comment]
I've stupidly deleted all the backups I have of the design reviews but they are all publicly listed on the Bitcoin Design Review channel. So will just upload to our new channel moving forward

**Reply by pavlenex** [reply]
Sounds good to me, but afaik you have a few options to download a video uploaded to YT. It's also supported natively within the dashboard.

**Comment by GBKS** [comment]
For completeness, here's a [link to the channel](https://www.youtube.com/channel/UCXKST1b8Wmq8Zt6b6S_-k3Q) we'll use.
